src/alignment.js

```diff
@@ -33,9 +33,13 @@ var getWalker = require('./walker').getWalker;
  * @return {boolean} True if the node matches, false otherwise.
  */
 var matches = function(node, nodeName, key) {
-  return node &&
-         key === getKey(node) &&
-         nodeName === getNodeName(node);
+  if (!node) {
```

Comment:
Perhaps we should make node non-nullable and just move this check to where it is called instead as it will end up being a bit shorter.